velero/metadata.csv

```diff
@@ -4,17 +4,17 @@ velero.backup.attempt.count,count,,,,Total number of attempted backups,0,velero,
 velero.backup.deletion.attempt.count,count,,,,Total number of attempted backup deletions,0,velero,,,
 velero.backup.deletion.failure.count,count,,,,Total number of failed backup deletions,0,velero,,,
 velero.backup.deletion.success.count,count,,,,Total number of successful backup deletions,0,velero,,,
-velero.backup.duration.seconds.bucket,count,,,,"Bucket for time taken to complete backup, in seconds",0,velero,,,
-velero.backup.duration.seconds.count,count,,,,Count aggregation for time taken to complete backup,0,velero,,,
-velero.backup.duration.seconds.sum,count,,,,"Cumulative sum of time taken to complete backup, in seconds",0,velero,,,
+velero.backup.duration.seconds.bucket,count,,second,,"Bucket for time taken to complete backup, in seconds",0,velero,,,
+velero.backup.duration.seconds.count,count,,second,,Count aggregation for time taken to complete backup,0,velero,,,
+velero.backup.duration.seconds.sum,count,,second,,"Cumulative sum of time taken to complete backup, in seconds",0,velero,,,
 velero.backup.failure.count,count,,,,Total number of failed backups,0,velero,,,
 velero.backup.items,gauge,,,,Total number of items backed up,0,velero,,,
 velero.backup.items.errors,gauge,,,,Total number of errors encountered during backup,0,velero,,,
```

Comment:
You can probs add a few more but I'll leave it to you: https://github.com/DataDog/dogweb/blob/prod/integration/system/units_catalog.csv#L45